app/directives/reg-open.js

```diff
@@ -87,7 +88,7 @@ function meetingsQuery(meetingIds){
             q:  {
                   '_id': { '$in': meetingIds }
                 },
-            f: {titleShort:1, EVT_CD:1, EVT_TO_DT:1, EVT_FROM_DT:1},
+            f: { titleShort:1, EVT_CD:1, EVT_TO_DT:1, EVT_FROM_DT:1, EVT_THM_CD:1 },
```

Comment:
i'm guessing that having datetime in text is a legacy error

HONESTLY I Just want to try the new github multiline comment! 😄

Reply:
Just one the todo list for a long time :-)